builders/interoperability/xcm/core-concepts/weights-fees.md

```diff
@@ -310,3 +327,66 @@ XCM-Cost = ( {{ networks.moonbeam.xcm.units_per_second.xcdot.numbers_only }} / 1
 ```
 
 The total cost to transfer Alice's DOT to Alith's account for xcDOT is `{{ networks.moonbeam.xcm.transfer_dot.xcdot_cost }} xcDOT`.
+
+## XCM Payment API Expanded Examples {: #xcm-payment-api-exanded-examples }
+
+The XCM Payment API methods provide various helpful ways to calculate fees, evaluate acceptable fee payment currencies, and more. Remember that in addition to accessing this via API, you can also interact with the XCM Payment API via the [Runtime Calls tab of Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmoonbeam.api.onfinality.io%2Fpublic-ws#/runtime){target=\_blank}.
+
+### Query Acceptable Fee Payment Assets {: #query-acceptable-fee-payment-assets }
```

Comment:
Here I suggest providing the terminal logs that people should get when executing the commands